core/lib/Cardano/Address/Script.hs

```diff
@@ -331,6 +370,10 @@ instance ToJSON Script where
         object ["any" .= fmap toJSON content]
     toJSON (RequireSomeOf count scripts) =
         object ["some" .= object ["at_least" .= count, "from" .= scripts]]
+    toJSON (ValidFromSlot slot) =
+        object ["valid_from" .= slot]
+    toJSON (ValidUntilSlot slot) =
+        object ["valid_until" .= slot]
```

Comment:
On a second though, perhaps **active** captures this better than **valid**. Because, the script is valid from the moment it is created, but it may not be active yet / anymore. :thinking: ?

Reply:
yes, `active` is better -> adopted in https://github.com/input-output-hk/cardano-addresses/pull/95/commits/c5f72743677b1e153d5cd8b510f4d60e8f58f9ba